### ISSUE TITLE
developing in stage 2.5th

### ISSUE THREAD

**Issue by jimpauly** [jimpauly/paulies-prediction-partners#84]
Critique where we may still be falling short of quality, accessibility for everyone. (not-overlabeling). Ive included screenshots of my phone on the site in desktop mode in portrait and landscaping mode. Some regions are missing. There should be no mobile version in any way yet. If my grandma is on her phone in landscape or portrait, all regions should be present.

Only the side bars and main region increase or decrease height. Only the header, nav, bottom bar and main region inscrease/decrease width and everything should scale with no wrapping or rearranging
